src/Compiler/Checking/TailCallChecks.fs

```diff
@@ -222,6 +222,13 @@ and CheckCall cenv args ctxts (tailCall: TailCall) =
             | Expr.App _ -> Some(TailCall.YesFromExpr cenv.g e)
             | IsAppInLambdaBody t -> Some t
             | _ -> None
+        | Expr.App(_funcExpr, _formalType, _typeArgs, args, _range) ->
```

Comment:
Can we use named arguments here :)

Reply:
Sure, sorry. Saw that the second when I pushed.